src/analyze-action.ts

```diff
@@ -183,7 +197,10 @@ async function run() {
   } catch (origError) {
     const error =
       origError instanceof Error ? origError : new Error(String(origError));
-    if (!actionsUtil.getOptionalInput("expect-error")) {
+    if (
+      actionsUtil.getOptionalInput("expect-error") === "false" ||
```

Comment:
Only accept `"true"`, all else is considered false.

```suggestion
      actionsUtil.getOptionalInput("expect-error") !== "true" ||
```

Reply:
Done, thank you!

Reply:
Done, thank you!